src/posix/platform/CMakeLists.txt

```diff
@@ -38,15 +38,15 @@ if(OT_POSIX_VIRTUAL_TIME)
     list(APPEND OT_PRIVATE_DEFINES "OPENTHREAD_POSIX_VIRTUAL_TIME=1")
 endif()
 
+set(OT_POSIX_CONFIG_RCP_BUS "UART" CACHE STRING "RCP bus type")
```

Comment:
according to suggestion from @abtink, we should make default value `""`, and only set when the variable is set `""`.